pygmsh/occ/geometry.py

```diff
@@ -55,6 +56,15 @@ def revolve(self, *args, **kwargs):
         assert angle < 2 * math.pi
         return super()._revolve(*args, **kwargs)
 
+    def add_boundary_layer(self, *args, **kwargs):
```

Comment:
Can't those go in common?

Reply:
This is the main purpose of my contribution. Personally, I used this to refine the boundary of a opencascade model

Reply:
If it's both occ and geo, it can as well go into common. Less code duplication.

Reply:
Good idea. Let me put this into common